### PR TITLE
Enable network ospf collection

### PR DESCRIPTION
**Pull request by rohitthakur2590**

### CODE REVIEW

**Review by rohitthakur2590 — COMMENTED** [review]
Need to update this for netqwork.ospf

File: changelogs/changelog.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+ancestor: null
```
Comment (rohitthakur2590) [inline changelogs/changelog.yaml:1]: need to update this  w.r.t current release

File: CHANGELOG.rst [hunk 2]
```diff
@@ -0,0 +1,47 @@
+====================================
+Network Bgp Collection Release Notes
```
Comment (ashwini-mhatre) [inline CHANGELOG.rst:2]: network ospf* instead of bgp

File: filter_plugins/health_check_view.py [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+OSPF_FULL_STATES = ["FULL/BDR", "FULL/DR"]
+
+def health_check_view(*args, **kwargs):
+    params = ["health_facts", "target"]
+    data = dict(zip(params, args))
+    data.update(kwargs)
+    if len(data) < 2:
+        raise AnsibleFilterError(
+            "Missing either 'health facts' or 'other value in filter input,"
+            "refer 'ansible.utils.health_check_view' filter plugin documentation for details",
```
Comment (ashwini-mhatre) [inline filter_plugins/health_check_view.py:170]: ansible.utils is invalid place

File: galaxy.yml [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+authors:
+  - Rohit Thakur (rohitthakur2590)
+dependencies:
+  "ansible.netcommon": ">=4.1.0"
+  "ansible.utils": ">=2.7.0"
+license_file: LICENSE
+name: bgp
+description: Ansible Network BGP enables user to manage the BGP resources independent of platforms and perform BGP health checks.
```
Comment (ashwini-mhatre) [inline galaxy.yml:9]: please change bgp to ospf

**Review by ashwini-mhatre — CHANGES_REQUESTED** [review]
please also include 'remediate' action if applicable

**Comment by rohitthakur2590** [comment]
> 

Ii is already implemented.

**Review by ashwini-mhatre — APPROVED** [review]

**Pull request merged**